src/data-types/binary.ts

```diff
@@ -26,7 +26,7 @@ const Binary: { maximumLength: number } & DataType = {
     return 'binary(' + length + ')';
   },
 
-  resolveLength: function(parameter) {
+  resolveLength: function(parameter): number {
```

Comment:
I don't think the return type needs to be repeated here. 🤔 

```suggestion
  resolveLength: function(parameter) {
```